pkg/controllers/manageagent/manageagent.go

```diff
@@ -109,6 +109,9 @@ func (h *handler) getAgentBundle(ns string, cluster *fleet.Cluster) (runtime.Obj
 		Spec: fleet.BundleSpec{
 			BundleDeploymentOptions: fleet.BundleDeploymentOptions{
 				DefaultNamespace: h.systemNamespace,
+				// https://stackoverflow.com/a/28818489 - create reference to bool
+				// Decrypt: true,
+				Decrypt: &[]bool{true}[0],
```

Comment:
@StrongMonkey: thoughts here? Looks good to me, but I have not seen this pattern before.

Reply:
Yeah It is fine, I always use this to create pointers too.